tests/run_tests.sh

```diff
@@ -0,0 +1,42 @@
+if [ "$#" -eq 0 ]; then
+    echo "Usage: $0 <argument>"
+    exit 1
+fi
+pip3 install -e lmi_utils/
```

Comment:
If the user does not need to install these package, it would better to have another terminal argument to achieve that